src/pages/Login.jsx

```diff
@@ -108,6 +116,9 @@ const Login = () => {
                                 aria-label="Password"
                                 autoComplete="off"
                             />
+                            <button type="button" onClick={handleTogglePassword}>
+                                {showPassword ? <FaEyeSlash color="white" /> : <FaEye color="white" />}
+                            </button>
```

Comment:
Could you use `FaEye`  one and apply the color via variable?

`<FaEye color={variable} />`

Reply:
Also, I just noticed that you have `<FaEyeSlash color="white" /> ` 2 times and the color is not diffrent

Reply:
Thank you for pointing that out. I'm new to open source contributions, and this is my first time contributing. I appreciate your feedback and will make sure to avoid this mistake in the future </>